presto-main/src/main/java/com/facebook/presto/operator/scalar/MapToMapCast.java

```diff
@@ -58,29 +57,23 @@ public static Block toMap(
             ConnectorSession session,
             @SqlType("map(FK,FV)") Block fromMap)
     {
-        // loop over all the parameter types and bind ConnectorSession if needed
-        // TODO: binding `ConnectorSession` should be done in function invocation framework
```

Comment:
We were previously assuming that `ConnectorSession` can be the `ith` parameter while with this change we assume that session is always the first parameter as we pass `session` as the first argument in the invoke calls below.